assets/src/Components/CloudLibrary/Library.js

```diff
@@ -335,6 +333,41 @@ const Library = ( {
 				</div>
 			</div>
 		);
+	};
+
+	if ( themeData !== false ) {
+		return (
+			<UpsellModal
+				title={ __( 'Coming soon' ) }
+				description={ sprintf(
+					// translators: %1$s: Theme Name %2$s Plugin Name.
+					__(
+						'Right now this feature is not available with your current setup. if you want to use it, you need to install %1$s theme and %2$s plugin',
+						'template-patterns-collection'
+					),
+					`<a href="${ themeURL }" target="_blank" rel="noreferrer">${ __(
```

Comment:
We should not translate theme and plugin name so no need to add __ before "Neve" and "Neve Pro Addon"